webpack.config.js

```diff
@@ -4,7 +4,10 @@ const webpack = require('webpack');
 const path = require('path');
 
 module.exports = {
-    entry: './static/src/javascripts/boot-webpack.js',
+    entry: [
+        './static/src/javascripts/boot.js',
+        './static/src/javascripts/bootstraps/standard/main.js',
```

Comment:
Webpack creates new chunk for `bootstraps/standard/main.js` when it hits the [`require()` call in boot.js](https://github.com/guardian/frontend/blob/166dcbf0c339ca1483ca15ba63fff8643bd75f7a/static/src/javascripts-legacy/boot.js#L43). Webpack treats the `require` call as a [split point](https://webpack.github.io/docs/code-splitting.html) for the application and would under normal circumstances load the chunk asynchronously.

However, since we're concatenating standard and the boot script into the same file, we don't want Webpack to treat this as a split point. Hence we bundle these modules together into the same entry chunk.

I think this makes sense for our use case.